ci/prow/make_config.go

```diff
@@ -590,6 +594,20 @@ func configureServiceAccountForJob(data *baseProwJobTemplateData) {
 	addVolumeToJob(data, "/etc/"+name, name, true)
 }
 
+// addExtraEnvVarsToJob adds any extra environment variables (defined on command-line) to a job.
+func addExtraEnvVarsToJob(data *baseProwJobTemplateData) {
+	if len(extraEnvVars) == 0 {
```

Comment:
would it be better check for nil? as it would be no-op if len(extraEnvvars) == 0 below, but nil would fail the job

Reply:
extraEnvVars won't be nil, because it's initialized as en empty slice. Removed the unnecessary check though.